.github/workflows/checklink.yaml

```diff
@@ -0,0 +1,14 @@
+name: Check Markdown links
+
+on: push
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: gaurav-nelson/github-action-markdown-link-check@v1
+      with:
+        use-quiet-mode: 'yes'
+        config-file: '.github/workflows/checklink_config.json'
```

Comment:
How about put configuration files into `.github`? And keep only put workflow files into `.github/workflows`.

Reply:
More repositories are placed in the same directory in [Real-life usage samples](https://github.com/gaurav-nelson/github-action-markdown-link-check#real-life-usage-samples), which is just two styles.

Reply:
I see. This is not a block comment. They're both ok.